Hi3Helper.Core/Lang/en_US.json

```diff
@@ -946,7 +962,10 @@
     "DownloadSettingsOption1": "Start game after install",
 
     "OpenInExternalBrowser": "Open in External Browser",
-    "CloseOverlay": "Close Overlay"
+    "CloseOverlay": "Close Overlay",
+    
+    "DbGenerateUid_Title": "Are you sure to change your user ID?",

```

Comment:
`Are you sure you wish to change your user ID?`

Reply:
Fixed